aspnetcore/release-notes/aspnetcore-9.0.md

```diff
@@ -29,6 +29,8 @@ This section describes new features for SignalR.
 
 [!INCLUDE[](~/release-notes/aspnetcore-9/includes/signalrActivities.md)]
 
+[!INCLUDE[](~/release-notes/aspnetcore-9/includes/signalr-trimming-aot.md)]
```

Comment:
@wadepickett  for now on add the include in WN only for testing, then copy/paste it into the include file and remove it so WN doesn't change.
cc @tdykstra